app/Http/Controllers/Api/LoginController.php

```diff
@@ -21,44 +21,43 @@ class LoginController extends ApiController
     public function login(LoginRequest $request)
     {
             $portalResponse = callAPIPortal($request);
-        if ($portalResponse->meta->code == Response::HTTP_OK) {
             $user = $this->saveUser($portalResponse, $request);
-                
-            return  response()->json([
-            'meta' => $portalResponse->meta,
-            'data' => $user,
-            ], Response::HTTP_OK);
+        if (isset($user['errors']['message'])) {
```

Comment:
Check cái này trước rồi mới `$user = $this->saveUser($portalResponse, $request);` nha.